services/movie.py

```diff
@@ -1,11 +1,13 @@
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        title: str = None,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]